### DISCUSSION TITLE
French Dataset

### DISCUSSION THREAD

**Discussion by FrischJulien** [coqui-ai/TTS#1264] [General Q&A]
Hello team,

First time posting, apologies if this question has been addressed somewhere else.
I am considering recording a French dataset, to use with VITS or whatever new model would be best.
There are some great tips about what makes a good dataset at https://github.com/coqui-ai/TTS/wiki/What-makes-a-good-TTS-dataset , but I still wonder if you create one from scratch :
- How many speakers
- How many hours of recording / sentences
The goal here is to enable TTS quality on par with IBM Watson, Azure Coginitive, etc...

Thanks!

**Comment by kin0303** [comment]
Hi, have you got the answer?

**Reply by FrischJulien** [reply]
More or less... If my understanding is correct, the general rule is to prefer a small number of speakers with as many hours of recording as possible. On French datasets specifically I have experimented with various database, with ok-ish results.
I'll try to do some web scrapping on youtube videos next.

**Comment by FrontierDK** [comment]
I am currently trying to do the exact same (other languages). Anything from the developers on the  number of speakers?
